include/mbedtls/config.h

```diff
@@ -275,6 +275,23 @@
 //#define MBEDTLS_SHA1_ALT
 //#define MBEDTLS_SHA256_ALT
 //#define MBEDTLS_SHA512_ALT
+
+/**
+ * \def MBEDTLS_ECP_MUL_ALT
+ *
+ * MBEDTLS_ECP_MUL_ALT:Uncomment a macro to let mbed TLS use your
+ * alternate API implementation of ECP.
+ * Keep in mind that the function prototypes should remain the same.
+ *
+ * Example: In case you uncomment MBEDTLS_ECP_MUL_ALT, 'mbedtls_ecp_mul' will
+ * perform its alternate implementation.
+ *
+ * Uncomment a macro to enable alternate implementation of the corresponding
+ * API.
```

Comment:
While not difficult to guess, the documentation should be explicit on which functions these options affect.